.circleci/config.yml

```diff
@@ -34,3 +71,39 @@ jobs:
           command: bash gradlew publishToMavenLocal
       - store_test_results:
           path: test-results
+
+  integrationtest:
+    working_directory: ~/amplify-android-home
+    docker:
+      - image: circleci/android@sha256:5cdc8626cc6f13efe5ed982cdcdb432b0472f8740fed8743a6461e025ad6cdfc
+    environment:
+      JVM_OPTS: -Xmx3200m
+    steps:
+      - checkout
+      - generate_gradle_wrapper
+      - configure_aws
+      - setup_emulator
+      - run:
+          name: Copy configuration files
+          command: |
+            aws s3 cp s3://amplify-circleci-android-payload/awsconfiguration.json  aws-analytics-pinpoint/src/androidTest/res/raw/awsconfiguration.json --profile amplify_sdk_test 
```

Comment:
Minor: Later, would be nice to have a map of source and destination and iterate over map and execute the `cp` command for each entry of the map.

Reply:
Yes. A future enhancement is to not have separate config files for each of the module but minimum possible number of config files. I'll create an issue with these enhancements.